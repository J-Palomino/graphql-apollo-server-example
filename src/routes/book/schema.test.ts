import { gql } from "apollo-server";
import { getTestServer } from "@server";

const { query } = getTestServer();

test("Books Query returns expected results", async () => {
  const res = await query({
    query: gql`
      query {
        books {
          title
          author
        }
      }
    `,
  });
  expect(res.data.books).toMatchSnapshot();
});
