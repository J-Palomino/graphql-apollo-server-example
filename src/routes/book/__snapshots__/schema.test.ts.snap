// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Books Query returns expected results 1`] = `
Array [
  Object {
    "author": "Kate Chopin",
    "title": "The Awakening",
  },
  Object {
    "author": "Paul Auster",
    "title": "City of Glass",
  },
]
`;
